test/multiple-render-targets.js

```diff
@@ -20,13 +20,13 @@ tape('multiple-render-targets', function (t) {
       notZero++
     }
   }
-  t.equals(notZero, 144000, `Only ${notZero} are not 0, expected 144000`)
+  t.equals(notZero, 15, `Only ${notZero} are not 0, expected 15`)
```

Comment:
Tests are now passing!

btw, what's the rationale/math behind `15`?

Reply:
fixed to be 16, sorry about that.